File: PS Multi Tools/Classes/ComboBoxProjectItem.vb
Imports System.ComponentModel

Public Class ComboBoxProjectItem

    Implements INotifyPropertyChanged

    Public Event PropertyChanged As PropertyChangedEventHandler Implements INotifyPropertyChanged.PropertyChanged

    Public Sub NotifyPropertyChanged(propName As String)
        RaiseEvent PropertyChanged(Me, New PropertyChangedEventArgs(propName))
    End Sub

    Private _ProjectFile As String
    Private _ProjectName As String

    Public Property ProjectName As String
        Get
            Return _ProjectName
        End Get
        Set
            _ProjectName = Value
            NotifyPropertyChanged("ProjectName")
        End Set
    End Property

    Public Property ProjectFile As String
        Get
            Return _ProjectFile
        End Get
        Set
            _ProjectFile = Value
            NotifyPropertyChanged("ProjectFile")
        End Set
    End Property

End Class

File: PSMultiTools/Classes/ComboBoxProjectItem.cs
using System.Collections.Generic;
using System.ComponentModel;
using System.Runtime.CompilerServices;

namespace PSMultiTools.Classes
{
    public class ComboBoxProjectItem : INotifyPropertyChanged
    {

        private string? _ProjectFile;
        private string? _ProjectName;

        public event PropertyChangedEventHandler? PropertyChanged;

        protected bool SetProperty<T>(ref T field, T value, [CallerMemberName] string? propertyName = null)
        {
            if (EqualityComparer<T>.Default.Equals(field, value))
                return false;

            field = value;
            OnPropertyChanged(propertyName);
            return true;
        }

        protected void OnPropertyChanged([CallerMemberName] string? propertyName = null)
        {
            PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(propertyName));
        }

        public string? ProjectName
        {
            get => _ProjectName;
            set => SetProperty(ref _ProjectName, value);
        }

        public string? ProjectFile
        {
            get => _ProjectFile;
            set => SetProperty(ref _ProjectFile, value);
        }

    }
}
